test/propagated-properties.html

```diff
@@ -296,16 +296,12 @@
           expect(dateTimePicker.min).to.equal('2019-09-01T08:00');
           expect(datePicker.min).to.equal('2019-09-01');
           expect(timePicker.min).to.equal('00:00:00.000');
-          dateTimePicker.value = '2019-09-01T12:00';
-          expect(timePicker.min).to.equal('08:00');
         });
 
         it('should have initial value for max', () => {
           expect(dateTimePicker.max).to.equal('2019-09-30T22:00');
           expect(datePicker.max).to.equal('2019-09-30');
           expect(timePicker.max).to.equal('23:59:59.999');
-          dateTimePicker.value = '2019-09-30T12:00';
-          expect(timePicker.max).to.equal('22:00');
```

Comment:
These removed parts of two tests were causing an issue in IE11 only because/when the field is "disabled".  No idea why, maybe IE11 has some special behaviour related to disabled attribute disabling updating some properties in children or some other weird side effect.

These tests were not supposed to modify the state anyway (see comment in "before" hook above) and this behaviour related to removed lines is already tested above under "Property propagation". These tests should be enough as they are now without the removed lines.